modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/form_portlet.js

```diff
@@ -558,6 +558,18 @@ AUI.add(
 						}
 					},
 
+					_fillRuleDraft: function() {
+						var instance = this;
+
+						var ruleBuilder = instance.get('ruleBuilder');
+
+						var ruleDraft = ruleBuilder.get('ruleDraft');
+
+						if (!ruleBuilder.ruleDraftEmpty(ruleDraft) || typeof ruleDraft == 'undefined') {
```

Comment:
Rename ruleBuilder.ruleDraftEmpty(ruleDraft) to ruleBuilder.**isRuleDraftEmpty**(ruleDraft)

Reply:
Please, group these expressions too 👍